linux_os/guide/system/software/integrity/fips/enable_fips_mode/oval/shared.xml

```diff
@@ -0,0 +1,25 @@
+<def-group oval_version="5.10">
+  <definition class="compliance" id="enable_fips_mode" version="1">
+    <metadata>
+      <title>Enable FIPS Mode</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 8</platform>
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>Check existence of /etc/system-fips and verify if FIPS is enabled in Linux kernel.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion test_ref="test_etc_system_fips" comment="check existence of /etc/system-fips" />
+      <extend_definition definition_ref="sysctl_runtime_crypto_fips_enabled" comment="check sysctl crypto.fips_enabled = 1" />
+      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
+    </criteria>
+  </definition>
+
+  <unix:file_test version="1" id="test_etc_system_fips" check="all" check_existence="all_exist" comment="/etc/system-fips exists">
+      <unix:object object_ref="object_etc_system_fips" />
+  </unix:file_test>
+
+  <unix:file_object version="1" id="object_etc_system_fips">
+      <unix:filepath>/etc/system-fips</unix:filepath>
+  </unix:file_object>
+</def-group>
```

Comment:
I expect to see all FIPS checks in this OVAL file. So, it needs to also check that the fips package is installed and that fips is enabled in grub as well.